maintainers/maintainer-list.nix

```diff
@@ -3200,9 +3200,7 @@
     github = "cageyv";
     githubId = 51059484;
     name = "Vladmir Samoylov";
-    keys = [
-      { fingerprint = "8916 F727 734E 77AB 437F  A33A 19AB 76F5 CEE1 1392"; }
-    ];
+    keys = [ { fingerprint = "8916 F727 734E 77AB 437F  A33A 19AB 76F5 CEE1 1392"; } ];
```

Comment:
Please don't touch unrelated lines

Reply:
This is the work of nixfmt and anyone that adds themselves to the contributors list may end up in this same situation if I end up undoing this change